modules/doc/content/getting_started/examples_and_tutorials/tutorial04_meshing/step03_workflow.md

```diff
@@ -0,0 +1,65 @@
+# Meshing Workflow in MOOSE
+
+## Constructing a Reactor Mesh Hierarchically
+
+MOOSE's Mesh System and Reactor Module contains numerous "MeshGenerator" objects which either (a) create a mesh from scratch or (b) perform operations on existing meshes. To create a mesh, the user must define a sequence of MeshGenerator object calls inside the `[Mesh]` block to construct a geometry beginning with the smallest components (pins) and building up to larger components (core). For example, after pins are defined, they can be patterned into an assembly, and assemblies can then be patterned into a core. Application of a peripheral core zone, trimming along symmetry lines and extrusion to 3D are optional in the final steps.
+
+The meshing workflow for a standard reactor core follows the general hierarchical process of identifying key features in the geometry and building them hierarchically in terms of smallest to largest (pins, assemblies, cores, core periphery):
+
+1. Define pins
+2. Define assembly by patterning existing pins into a lattice and adding coolant background and/or duct region
+3. Define core by patterning assemblies into a lattice
+4. Apply core periphery zones like a circular shield
+5. Trim along lines of symmetry to reduce computational expense
+6. Extrude to 3D
+
+## Execution
+
+Use any executable with the Reactor module compiled (see [step02_building.md]), such as `$MOOSE_DIR/modules/combined/combined-opt` or `$MOOSE_DIR/modules/reactor-opt` along with the `--mesh-only` command line option:
+
+```bash
+$MOOSE_DIR/modules/reactor/reactor-opt -i reactormesh.i --mesh-only
```

Comment:
I would either point to a real input or make it explicit like this `<my_meshing_input.i>`